### PR TITLE
Test TransactionView

### PR DESCRIPTION
**Pull request by johnedma**
Soundtrack of this PR: [Alan Watts: Silence is key](https://www.youtube.com/watch?v=rJwVlmlyhqc)

### Motivation

Continued test coverage expansion for Desktop Wallet components, views, utils and hooks.

### In this PR

- Tests snapshot of component matches
- Verify `SendMobPanel` renders
- Verify `ReceiveMobPanel` renders
- Tabs update panel views on click 

### Testing
PASS  test/views/wallet/TransactionView/TransactionView.test.tsx (8.707 s)

File                  | % Stmts | % Branch | % Funcs | % Lines | Uncovered Line #s
----------------------|---------|----------|---------|---------|-------------------
  index.tsx                                    |     100 |      100 |     100 |     100 |                                                                               

Test Suites: 1 passed, 1 total
Tests:       3 passed, 3 total
Snapshots:   1 passed, 1 total
Time:        14.264 s

### CODE REVIEW

File: test/views/wallet/TransactionView/TransactionView.test.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+      mobUrl: 'string',
+    };
+  });
+
+  const { asFragment } = renderSnapshot(
+    <TransactionView />,
+  );
+
+  const sendQuery = screen.queryByText(/Please enter the amount of MOB you want to send and the public address of the recipient./i);
+  const receiveQuery = screen.queryByText(/To receive MOB, you must share your public address code to the sender./i);
```
Comment (fktt22) [inline test/views/wallet/TransactionView/TransactionView.test.tsx:31]: Just a question: Is the final /.../i actually needed?

File: test/views/wallet/TransactionView/TransactionView.test.tsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        const { sendQuery, receiveQuery } = setupComponent();
+        userEvent.click(screen.getByText('Send MOB'));
+        expect(sendQuery).toBeInTheDocument();
+        expect(receiveQuery).not.toBeInTheDocument();
+      });
+
+      test('ReceiveMobPanel renders correctly', () => {
+        const { sendQuery } = setupComponent();
+        userEvent.click(screen.getByText('Receive MOB'));
+        expect(screen.queryByText(/To receive MOB, you must share your public address code to the sender./i)).toBeInTheDocument();
```
Comment (fktt22) [inline test/views/wallet/TransactionView/TransactionView.test.tsx:58]: Same question about the `/.../i` regex.

File: test/views/wallet/TransactionView/TransactionView.test.tsx (hunk 1 above)
Comment (johnedma) [reply test/views/wallet/TransactionView/TransactionView.test.tsx:31]: It wasn't, was still beginning with testing library so tried querying different ways.

**Review by davidyhall — APPROVED** [review]
LGTM! I'm not surprised that line 58 was allowed by the linter, the test linter configurations are all wrong right now. We'll clean that up on another PR.

**Comment by davidyhall** [comment]
Here's a helpful guide to the difference of string and regex patterns for text matching. https://testing-library.com/docs/queries/about/#textmatch -- I suspect you actually mean to just mean to do a `full string match`, and at that point, you might as well just use single quotes. But! this is a nit imo and I won't block this.

**Pull request merged**